server/index.js

```diff
@@ -180,4 +181,24 @@ app.get('/api/your/fitness', (req, res, next) => {
     }).catch(err => next(err));
 });
 
+app.post('/api/auth/register', (req, res, next) => {
+  const { username, firstName, lastName, password } = req.body;
+  if (!username || !password || !firstName || !lastName) {
+    throw new ClientError(400, 'You missed something!');
```

Comment:
This error message sucks. Try to give more specific information about what the client developer needs to fix about their request.